db/schema.rb

```diff
@@ -1615,8 +1617,13 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.integer "version"
-    t.integer "version_id"
+    t.bigint "version_id"
     t.index ["version", "zip_code"], name: "zipcode_rates_archives_version_zip_code_idx"
+    t.index ["version_id"], name: "index_zipcode_rates_archives_on_version_id"
   end
 
+  add_foreign_key "institutions", "versions"
+  add_foreign_key "institutions_archives", "versions"
+  add_foreign_key "zipcode_rates", "versions"
+  add_foreign_key "zipcode_rates_archives", "versions"
```

Comment:
Do these lines belong in `schema.rb`?

Reply:
Hi @jholton 

These lines get added to `schema.rb` upon executing the database migration script after making your recommended changes regarding indexing and foreign key constraints for this pull request.

Reply:
You are correct!  My mistake.  I got confused because you can also use the `add_foreign_key` method in migrations.